android/shopify_buy_plugin/build.gradle

```diff
@@ -28,6 +28,7 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile group: 'com.google.code.gson', name: 'gson', version: '2.3.1'
```

Comment:
Any reason in particular you're pulling in Gson? Android has a `JSONObject` class that you can use to do simple JSON serializing/deserializing.